Binance.Net/Objects/Shared/BinanceOrderBase.cs

```diff
@@ -129,19 +129,5 @@ public decimal Price
         /// Quantity which is still open to be filled
         /// </summary>
         public decimal QuantityRemaining => Quantity - QuantityFilled;
-
-        /// <summary>
-        /// The average price the order was filled
-        /// </summary>
-        public decimal? AverageFillPrice
```

Comment:
Why was this removed?

Reply:
The Price property is also returned by Binance, and if I'm not mistaken it's the original placement price, not necessarily the fill price. So both properties offer different info. `_price == 0 && Type == OrderType.Market` was checked in the Price property because market orders don't have a price, so the property just returns the fill price.